lib/service/database.js

```diff
@@ -68,26 +68,41 @@ var affectedTableRegexCache = {
 
 DatabaseService.prototype.getLastUpdatedTimeFromAffectedTables = function(node, skip, callback) {
```

Comment:
I thought about it but from my point of view we're returning last updated time from affected tables in the form of last_updated time and the affected table for each time entry. Don't you think?